### PR TITLE
add product listing page

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
Looks good to me

**Review by tewarimohit — APPROVED** [review]
Looks good to merge...

File: pages/product-listing-page/products..css [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+.products-menu{
+    display: grid;
+    justify-content: center;
+    grid-template-columns: repeat(auto-fit, minmax(270px, 1fr));
+    gap: 1rem;
+}
+
+.card-component{
+    max-width: 350px;
```
Comment (Jia-A) [inline pages/product-listing-page/products..css:68]: You can change this value to rem from px.

File: pages/product-listing-page/products..css [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+}
+
+.card-wrapper {
+    border: 1px solid var(--primary-color);
+    background-color: inherit;
+    box-shadow:  2px 2px 5px 3px var(--off-white);
+    cursor: pointer;
+    display: flex;
+    flex-direction: column;
+    max-width: 350px;
```
Comment (Jia-A) [inline pages/product-listing-page/products..css:79]: Here too px to rem.

**Review by Jia-A — COMMENTED** [review]
Just change px to rems in two mentioned places. Everything else looks good.

File: pages/product-listing-page/products..css (hunk 2 above)
Comment (ganesh-kanchi) [reply pages/product-listing-page/products..css:79]: ok, will do.

**Pull request merged**